panics.go

```diff
@@ -163,6 +163,20 @@ func Capture(err string, message ...string) {
 	publishError(errors.New(err), []byte(tmp), false)
 }
 
+// Capture will publish any errors with stack trace
+func CaptureWithStackTrace(err string, message ...string) {
+	var tmp string
+	for i, val := range message {
+		if i == 0 {
+			tmp += val
+		} else {
+			tmp += fmt.Sprintf("\n\n%s", val)
```

Comment:
Do we have some test regarding the formatting?

Reply:
![Screenshot from 2019-04-02 11-27-57](https://user-images.githubusercontent.com/8907846/55376399-73c0f380-553a-11e9-9ec5-8f6e025fbe1a.png)
This is generated from this function. Btw, this function already exists, only copy pasted, with minor flag change to publishError

Reply:
Hi Davin, looking good. But do you have some formatting test on your side? For example, you might expect some format when printing the output. 

Maybe either you validate the format from the value itself or using a writer to validate the format from logs/shell.

This package is lacking test(this is also my bad), that is why I am asking to do so.

Reply:
hello Albert, I added some tests ya. I used file loggings to do formatting assertions from the tests

Reply:
LGTM, thanks Davin.